release-tools/build.make

```diff
@@ -57,12 +57,30 @@ else
 TESTARGS =
 endif
 
+BUILD_IN_DOCKER?=false
+DOCKER_BUILDER_WD=/go/src/github.com/kubernetes-csi/external-attacher
+DOCKER_BUILDER_IMG=golang
+
+vendor: Gopkg.toml Gopkg.lock
```

Comment:
Better don't assume that the vendor directory gets managed by dep. We are about to change that.

Why is this necessary? Can't you make the existing "vendor" directory available inside the Docker container?

Reply:
I just thought being able to run `dep` in container might be usable for folks that need to perform the whole build process (CI) in e.g Debian stable. I'm cool to drop this part though.